### PR TITLE
Update just-get to handle accessing properties on null/undefined

### PR DESCRIPTION
**Pull request by arkadyp**
I was running into a lot of JS error using just-get alongside [reselect](https://github.com/reactjs/reselect) due to trying to access properties on `null`/`undefined`.

I think that it makes sense for just-safe-get to follow the same API as dotty (which is used in a lot of Uber web projects) where the library returns undefined when accessing properties on these values instead of throwing a JS error.

### CODE REVIEW

File: packages/object-safe-get/index.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ module.exports = get;
 */
 
 function get(obj, props) {
+  if (!obj) {
+    return undefined;
```
Comment (angus-c) [inline packages/object-safe-get/index.js]: maybe we should return obj instead (so if obj is null it returns null). This is also consistent with the logic for sub-properties

File: test/object-safe-get/index.js [hunk 2]
```diff
@@ -66,6 +66,13 @@ test('returns undefined for falsey property names using array arg', function(t)
   t.end();
 });
 
+test('returns undefined when object is null or undefined', function(t) {
+  t.plan(2);
+  t.ok(compare(get(null, 'a')), undefined);
```
Comment (angus-c) [inline test/object-safe-get/index.js]: this needs to be null now

File: README.md [hunk 3]
```diff
@@ -548,6 +548,9 @@ get(obj.a, ['aa', 'aaa']); // 2
 get(obj.b, 'bb.bbb'); // undefined
 get(obj.b, ['bb', 'bbb']); // undefined
 
+get(null, 'a') // undefined
```
Comment (angus-c) [inline README.md]: should be null

File: packages/object-safe-get/README.md [hunk 4]
```diff
@@ -22,6 +22,9 @@ get(obj.a, ['aa', 'aaa']); // 2
 get(obj.b, 'bb.bbb'); // undefined
 get(obj.b, ['bb', 'bbb']); // undefined
 
+get(null, 'a') // undefined
```
Comment (angus-c) [inline packages/object-safe-get/README.md]: should be null

**Comment by angus-c** [comment]
thx! will publish tonight

**Comment by arkadyp** [comment]
Awesome thank you!!

**Comment by angus-c** [comment]
published!

**Pull request merged**